app/gateway/kong-manager.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Kong Manager"
+content_type: reference
+layout: reference
+
+products:
+    - gateway
+
+min_version:
+  gateway: '3.4'
+
+description: Kong Manager is the graphical user interface (GUI) for {{site.base_gateway}}.
+
+faqs:
+  - q: I can't access Kong Manager. How do I fix a Kong Manager URL that doesn’t resolve?
+    a: |
+      Most likely, the port wasn't exposed during installation. Install a new instance and map port `8002` during installation.
+      
+      For example, with a Docker install:
+
+      ```
+      -p 127.0.0.1:8002:8002
+      ```
+
+related_resources:
+  - text: Enable Basic Auth for Kong Manager
+    url: /how-to/enable-basic-auth-on-kong-manager/
+  - text: Enable OIDC for Kong Manager
+    url: /how-to/enable-oidc-for-kong-manager/
+  - text: Set up authenticated group mapping in Kong Manager with OIDC
+    url: /how-to/oidc-authenticated-group-mapping/
+  - text: Configure LDAP with Kong Manager
+    url: /how-to/configure-ldap-with-kong-manager/
+  - text: Configuring Kong Manager to Send Email
+    url: /how-to/configure-kong-manager-email/
+---
+
+@todo
+
+Pull content from https://docs.konghq.com/gateway/latest/kong-manager/
```

Comment:
```suggestion
```
I looked at that page, and I think most of that content will go on the Kong Gateway landing page - if it even needs to exist (eg, Kong OSS isn't a thing anymore so we won't have a comparison table)